examples/plugins/cli-plugin/README.md

```diff
@@ -55,24 +52,24 @@ You can query the list of installed plugins:
 
     $ singularity plugin list
     ENABLED  NAME
-        yes  sylabs.io/test-plugin
+        yes  sylabs.io/cli-plugin
 
 Disable an installed plugin:
 
-    $ singularity plugin disable sylabs.io/test-plugin
+    $ singularity plugin disable sylabs.io/cli-plugin
 
 Enable a disabled plugin:
 
-    $ singularity plugin enable sylabs.io/test-plugin
+    $ singularity plugin enable sylabs.io/cli-plugin
 
 Uninstall an installed plugin:
 
-    $ singularity plugin uninstall sylabs.io/test-plugin
+    $ singularity plugin uninstall sylabs.io/cli-plugin
 
 And inspect a SIF file before installing:
 
-    $ singularity plugin inspect examples/plugins/test-plugin/test-plugin.sif
-    Name: sylabs.io/test-plugin
-    Description: This is a short test plugin for Singularity
-    Author: Michael Bauer
+    $ singularity plugin inspect examples/plugins/cli-plugin/cli-plugin.sif
+    Name: sylabs.io/cli-plugin
+    Description: This is a short test CLI plugin for Singularity
+    Author: Sylabs Team
     Version: 0.0.1
```

Comment:
Again, as per [this semver recommendation](https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase), should be `0.1.0`.